src/frontends/tensorflow_common/include/common_op_table.hpp

```diff
@@ -0,0 +1,149 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <functional>
+#include <map>
+#include <string>
+
+#include "openvino/core/node_vector.hpp"
+#include "openvino/frontend/node_context.hpp"
+#include "openvino_conversions.hpp"
+#include "utils.hpp"
+
+#define TENSORFLOW_OP_VALIDATION(node_context, ...)                                        \
+    OPENVINO_ASSERT_HELPER(::ov::frontend::OpValidationFailure,                            \
+                           ("While validating node '" + node_context.get_op_type() + "'"), \
+                           __VA_ARGS__)
+
+using namespace std;
```

Comment:
Please do not use using in headers

Reply:
Removed. BTW it was taken as-is from TF FE